src/plugins/BrowserStorageFacade.ts

```diff
@@ -0,0 +1,88 @@
+import { EncryptedKey } from "../types";
+
+export interface BrowserStorageConfigParams {
+  prefix?: string;
+  storage: any;
```

Comment:
```suggestion
  storage: Storage;
```

Reply:
Ah, good catch - I forgot to add this interface. It's actually not `Storage`, which is the interface for localStorage. I added it using the spec here: https://developer.chrome.com/docs/extensions/reference/storage/